doc/classes/XRServer.xml

```diff
@@ -108,6 +108,10 @@
 		<member name="primary_interface" type="XRInterface" setter="set_primary_interface" getter="get_primary_interface">
 			The primary [XRInterface] currently bound to the [XRServer].
 		</member>
+		<member name="world_origin" type="Transform3D" setter="set_world_origin" getter="get_world_origin" default="Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0)">
+			The current origin of our tracking space in the virtual world. This is used by the renderer to properly position the camera with new tracking data.
+			[b]Note:[/b] This property is managed by the current [XROrigin3D] node. It is exposed for access from GDExternal plugins.
```

Comment:
"GDExternal" isn't a thing :D

Reply:
OK I'm too tired, it both starts with an E :P